controlinterface/views.py

```diff
@@ -217,6 +218,23 @@ def subscription_edit(request):
                     "confirmbabyform": confirmbabyform,
                 })
                 context.update(csrf(request))
+    elif request.method == "GET" and 'msisdn' in request.GET:
+        # Confirm before updating the record
+        msisdn = request.GET.get("msisdn")
+
```

Comment:
We need to validate this msisdn has subs. If not, switch to just an optout button. For consistency please also add this to the current lookup screen.